PKGBUILD

```diff
@@ -20,6 +20,14 @@ ros_makedepends=(
   ros-melodic-nav-msgs
   ros-melodic-realtime-tools
   ros-melodic-pluginlib
+# test depends
+  ros-melodic-controller-manager
```

Comment:
Could they be marked as `testdepends`?

Reply:
If that exists, this might be the better approach. Also, together with https://github.com/ros-melodic-arch/ros-melodic-ackermann-steering-controller/pull/2#issuecomment-683405016